src/vscode-dts/vscode.proposed.taskProblemMatcherStatus.d.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'vscode' {
+
+
+	export interface TaskProblemMatcherStartedEvent {
+		/**
+		 * The task item representing the task for which the problem matcher processing started.
+		 */
+		readonly execution: TaskExecution;
+	}
+
+	export interface TaskProblemMatcherEndedEvent {
+		/**
+		 * The task item representing the task for which the problem matcher processing ended.
+		 */
+		readonly execution: TaskExecution;
+
+		/**
+		 * Whether errors were found during the task execution
+		 */
+		readonly hasErrors: boolean;
```

Comment:
Does this mean errors while processing or diagnostics (aka squiggles) created and iff so is this limited to error diagnostics or any kind of them?

Reply:
Good question - this is only for errors.